hail/src/main/scala/is/hail/variant/ReferenceGenome.scala

```diff
@@ -381,9 +378,10 @@ case class ReferenceGenome(name: String, contigs: Array[String], lengths: Map[St
     fastaReaderCfg = null
   }
 
-  private[this] var liftoverMaps: Map[String, LiftOver] = Map.empty[String, LiftOver]
+  private var chainFiles: Map[String, String] = Map.empty
+  @transient private[this] var liftoverMap: mutable.Map[String, LiftOver] = _
 
-  def hasLiftover(destRGName: String): Boolean = liftoverMaps.contains(destRGName)
+  def hasLiftover(destRGName: String): Boolean = liftoverMap.contains(destRGName)
```

Comment:
this should look at chain files, not liftoverMap, right?

Reply:
they should be equivalent, but done

Reply:
I think there was a NPE in a test coming from here, because after deserialization when heal hasn't been run, this is incorrect. Maybe that's a usage we don't care to test though. Regardless, the chain files seems like the correct source of truth though.